### PR TITLE
Biomass

### PR DESCRIPTION
**Pull request by lukelaujaa**
Biomass functions by Repola for pine, spruce and birch.

### CODE REVIEW

File: tests/forestry/forestry_biomass_test.py [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+        print("stem, waste with bark:")
+        print(round(sum(stem_waste_with_bark),1))
+        print("living branches")
+        print(round(sum(living_branches),1))
+        print("dead branches")
+        print(round(sum(dead_branches),1))
+        print("kanto")
+        print(round(sum(stump),1))
+        print("juuret 2mm")
+        print(round(sum(coarse_roots),1))
```
Comment (jpy-luke) [inline tests/forestry/forestry_biomass_test.py:254]: print statements for debug purposes shouldn't be left in the released code

File: forestry/biomass_repola.py [hunk 2]
```diff
@@ -0,0 +1,429 @@
+import math
+import pandas as pd
+import volume_ak as vol
+from ForestDataModels import ForestStand,ReferenceTree
+
+#from forestry.ForestDataModels import ForestStand
```
Comment (jpy-luke) [inline forestry/biomass_repola.py]: Could remove this line

File: forestry/biomass_repola.py [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+#Biomass components 
+#1. Stem with bark
+#2. Stem roundwood with bark
+#3. Stem waste with bark
+#4. Living branches
+#5. Dead branches
+#6. Needles / Foliage
+#7. Stump
+#8. Roots > 1 cm
```
Comment (jpy-luke) [inline forestry/biomass_repola.py:27]: This could be made as a new class in the future, then the functions would return objects of that class.

Comment (jpy-luke) [reply forestry/biomass_repola.py:27]: This does not necessarily need to be done now.

File: tests/forestry/forestry_biomass_test.py (hunk 1 above)
Comment (lukelaujaa) [reply tests/forestry/forestry_biomass_test.py]: done

File: tests/forestry/forestry_biomass_test.py [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+        print("stem, bark:")
+        print(round(sum(stem_bark),1))
+        print("living branches")
+        print(round(sum(living_branches),1))
+        print("dead branches")
+        print(round(sum(dead_branches),1))
+        print("kanto")
+        print(round(sum(stump),1))
+        print("juuret 2mm")
+        print(round(sum(coarse_roots),1))
```
[the remark this answers is not in the document]
Comment (lukelaujaa) [reply tests/forestry/forestry_biomass_test.py]: done

**Review by jpy-luke — APPROVED** [review]

**Pull request merged**